docs/reference/generated/number-field-root.json

```
@@ -11,15 +11,15 @@
       "description": "The uncontrolled value of the field when it’s initially rendered.\n\nTo render a controlled number field, use the `value` prop instead."
     },
     "value": {
-      "type": "number",
+      "type": "null | number",
```

```suggestion
      "type": "number | null",
```

It's strate null to be the first option. Do you know why it was generated this way? The prop is defined as `value?: number | null;`